portfolio/src/main/webapp/style.css

```diff
@@ -11,7 +11,7 @@ body {
 }
 
 #fullpage {
-  min-width: 100vw;
+min-width: 100vw;
```

Comment:
indent by 2 spaces and maintain sorted declaration order

Reply:
Updated with commits df04000 and 6581af5